src/translate.py

```diff
@@ -3231,12 +3459,31 @@ def reg_at(index: int) -> List[Register]:
         return arch.all_return_regs
```

Comment:
the `jal` handling should include `bl` as well, to match the treatment in the main translation function

Reply:
also, in the
```
                for reg, val in arch.function_return(call):
                    ...
```
loop, we should assert that it loops over everything in `arch.all_return_regs`, to ensure output_regs_for_instr is always correct